src/Apps/Artwork/Components/ArtworkSidebar2/ArtworkSidebar2Classification.tsx

```diff
@@ -28,11 +28,11 @@ const ArtworkSidebar2Classification: React.FC<ArtworkSidebar2ClassificationProps
 
   const closeModal = () => setIsModalOpen(false)
 
-  if (!artwork.attributionClass) {
+  if (!artwork.attributionClass?.shortArrayDescription?.length) {
```

Comment:
**suggestion**: `if (artwork.attributionClass?.shortArrayDescription?.length === 0) {...}`

Reply:
what do you think of something like this:
```suggestion
  if (artwork.attributionClass?.shortArrayDescription?.length !== 0) {
```